src/platforms/hosted/bmp_remote.c

```diff
@@ -46,6 +46,7 @@ int remote_init(void)
 		return -1;
 	}
 	DEBUG_PROBE("Remote is %s\n", construct + 1);
+	remote_target_set_power(power_up) ;
```

Comment:
Please remote the extra space between `)` and `;` (this will be the PR lint check issue)